### ISSUE TITLE
Stats: Adding all visitors from SiteVisitStats can create inaccurate total visitor count

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-ios#8173]
**Describe the bug**

Right now, we're computing a site's total visitor count in the `SiteVisitStats` model by adding together the visitor count for each of the intervals within a given period:

https://github.com/woocommerce/woocommerce-ios/blob/835f9c6abad4cddc848835c8ebee668e83111af8/Networking/Networking/Model/Stats/SiteVisitStats.swift#L43-L48

However, calculating the total visitors this way can be inaccurate. As outlined in [this support doc](https://wordpress.com/support/stats/#views-and-visitors):

> - A view is counted when a visitor loads or reloads a page.
> - A visitor is counted when we see a user or browser for the first time in a selected time frame.

In other words:

If I’m viewing my stats for today, I might see that I’ve had 3 visitors today. I might have 3 visitors every day for a week. However, my total visitors for a week **won’t** necessarily be a sum of the visitors I had for each day of that week (3 + 3 + 3 + 3 + 3 + 3 + 3 = 21 visitors). If I had 3 visitors every day, but they were the _same_ 3 visitors, my total visitor count for the week should be 3 (instead of 21); this is the total we would get if we retrieved the total visitor count with the `week` granularity.

See also this internal discussion: pe5pgL-1eR-p2#comment-1286, pe5pgL-1p6-p2

**Comment by rachelmcr** [comment]
This issue can account for discrepancies between the visitor counts in different selected time periods on the My Store dashboard (see https://github.com/woocommerce/woocommerce-ios/issues/1795), and if we reuse the same method for getting visitor counts in the Analytics Hub we'll be reproducing the issue there.

**Comment by rachelmcr** [comment]
I think the main fix for this will be to add a second API request for getting visitor stats:

1. We can use the current request to `/sites/{SITE}/stats/visits` to get the visitor stats for each interval (used for the chart when a single interval is selected).
2. We can add a request to `/sites/{SITE}/stats/summary` to get the total visitor stats for the entire time period (displayed when there are no intervals selected on the chart).
